app/handlers/api.py

```diff
@@ -45,6 +45,108 @@
 from os.path import splitext
 
 
+class AutoCropperHandler(BaseHandler):
+    SUPPORTED_METHODS = ("GET", "POST")
+
+    @asynchronous
+    @engine
+    @web_authenticated
+    def get(self,):
+        response = yield Task(
+            self.api_call,
+            url=self.settings['API_URL'] + '/autocropper',
+            method='GET')
+        self.set_json_output()
+        if response.code == 404:
+            body = loads(response.body.decode("utf-8"))
+            self.finish(self.json_encode({
+                        'status': 'success',
+                        'message': body['message'],
+                        'data': []}))
+        else:
+            self.set_status(response.code)
```

Comment:
Could we implement error handling in the get method to check if the expected fields, like 'code', 'message' exist in the response body before accessing them to avoid potential issues?